tools/test-proxy/Azure.Sdk.Tools.TestProxy.Tests/IntegrationTests/GitStoreIntegrationPushTests.cs

```diff
@@ -363,5 +361,81 @@ public async Task Scenario4(string inputJson)
                 TestHelpers.CleanupIntegrationTestTag(updatedAssets);
             }
         }
+
+        /// <summary>
+        /// 1. Restore from a tag that has minimal existing files and a long destination path
+        /// 2. Add a _bunch_ of files
+        /// 3. Push
+        /// 4. Verify local files are what is expected
+        /// 5. Verify assets.json was updated with the new commit Tag
+        /// </summary>
+        /// <param name="inputJson"></param>
+        /// <returns></returns>
+        [EnvironmentConditionalSkipTheory]
+        [InlineData(1000, 0.15)] // 1000 0.15 MB files
+        [InlineData(150, 0.25)]  // 150  0.25 MB files
+        [InlineData(100, 0.50)]  // 100  0.5 MB files
+        [InlineData(10, 1.50)]   // 10   1.5 MB files
+        [InlineData(1, 1024)]    // 11   1 GB file
```

Comment:
You can skip the GB file, git will complain if you try and push anything over 100 MB.

Reply:
You're absolutely right and we removed it.